test/test_loader.py

```diff
@@ -314,7 +314,7 @@ def test_clean_callable_default(tmpconfig, func):
     assert not hasattr(func, 'commands')
     assert not hasattr(func, 'nickname_commands')
     assert not hasattr(func, 'action_commands')
-    assert not hasattr(func, 'intents')
+    assert not hasattr(func, 'ctcp')
```

Comment:
What, you don't want to make sure nothing is adding the old `intents` attr? 😁

Reply:
Indeed, I don't care. 😁